app/org/sagebionetworks/bridge/models/accounts/User.java

```diff
@@ -30,16 +35,18 @@
     private String email;
     private String healthCode;
     private String studyKey;
-    private boolean consent;
-    private boolean signedMostRecentConsent;
     private SharingScope sharingScope;
     private Set<Roles> roles = Sets.newHashSet();
     private Set<String> dataGroups = Sets.newHashSet();
+    @JsonDeserialize(keyUsing = SubpopulationGuidDeserializer.class)
+    private Map<SubpopulationGuid,ConsentStatus> consentStatuses;
```

Comment:
No ConsentStatus doesn't contain healthCode, it's a summary of the status of consent to that consent. It only has the subpopulation guid. SubpopulationGuidDeserializerTest tests the serialization of this map.